packages/manager/apps/pci-instances/src/data/hooks/image/selector/image.selector.spec.tsx

```diff
@@ -10,12 +10,25 @@ const t: TFunction = ((key: string, options?: { name: string }) => {
   return key;
 }) as TFunction;
 
+const fakeImage: TImageDto = {
+  name: 'Ubuntu',
+  osType: '',
+  baseImageRef: '',
+  createdAt: '',
+  id: 'foo',
+  region: '',
+  size: 0,
+  status: 'active',
+  type: 'official',
```

Comment:
just a detail: previous ones are "standard" type, I don't know if it's important or not here

Reply:
`standard` types didn't exist in the original code..

Here is the correct signature of the type :)
```ts
export type TImageType = 'custom' | 'official' | 'rescue' | 'snapshot';
```